src/components/List/ListHeader/ListHeader.jsx

```diff
@@ -45,6 +45,7 @@ const ListHeader = ({ title, buttons, search, i18n, isLoading }) => {
       {search && !isLoading ? (
         <div className={`${iotPrefix}--list-header--search`}>
           <Search
+            id={`${iotPrefix}--list-header--search`}
```

Comment:
What's this one for?

Reply:
If the id isn't set, a random id is produced on every render, causing the snapshot to never be the same